kotlin/lib/src/main/kotlin/Application.kt

```diff
@@ -0,0 +1,56 @@
+package com.svix.kotlin
+
+import com.svix.kotlin.internal.apis.ApplicationApi
+import com.svix.kotlin.internal.infrastructure.ServerException
+import com.svix.kotlin.internal.models.ApplicationIn
+import com.svix.kotlin.internal.models.ApplicationOut
+import com.svix.kotlin.internal.models.ListResponseApplicationOut
```

Comment:
looking great so far, one concern i have is that the models are under `com.svix.kotlin.internal` in java we had the ability to move the models to a separate package like `com.svix.kotlin.models` via an openapi-generator config option but looking at the [kotlin generator](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/kotlin.md) I'm not seeing a similar option.  @talhacohen is there a way in kotlin to re-export the models package a level up? (otherwise we might need to get our hands dirty with the templates.

Reply:
Nope, the bindings that @talhacohen is working on look good so far, but this will be a blocker for release

Reply:
Done

Reply:
🙌 nice! The documentation I was looking at didn't have this option listed for jolting

Reply:
Yeah I couldn't find that either, I saw in https://openapi-generator.tech/docs/usage/#generate that it's supported as CLI flag, so just tried it out in the config and it works :)